config/mnemosyne.yml.example

```diff
@@ -0,0 +1,26 @@
+# Mnemosyne AMQP sink
```

Comment:
What about copying the `mnemosyne.yml.example` for the Vagrant setup? If you do so, please ensure `mnemosyne.yml` is in `.gitignore`